server/controllers/auth.js

```diff
@@ -35,7 +35,7 @@ function loginRoute(req, res, next) {
     .then(session => {
       // bind the session variables
       _.merge(req.session, session);
-      session.token = JWTConfig.create(session);
+      session.token = JWTConfig.create({ ...session.user });
```

Comment:
Oh, that's good but otherwise we only create the information of the user object, this will make it easier after decoding to reload the information in the session.
file `server/config/access.js` line 26